packages/PackageBuilder/src/DependencyInjection/CompilerPass/ConfigurableCollectorCompilerPass.php

```diff
@@ -0,0 +1,69 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\PackageBuilder\DependencyInjection\CompilerPass;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symplify\PackageBuilder\DependencyInjection\DefinitionCollector;
+use Symplify\PackageBuilder\DependencyInjection\DefinitionFinder;
+
+final class ConfigurableCollectorCompilerPass implements CompilerPassInterface
+{
+    /**
+     * @var DefinitionCollector
+     */
+    private $definitionCollector;
+
+    /**
+     * @var string[][]
+     */
+    private $defaultCollectors = [
+        # symfony/console
+        [
+            'main_type' => 'Symfony\Component\Console\Application',
+            'collected_type' => 'Symfony\Component\Console\Command\Command',
+            'add_method' => 'add',
+        ],
+        # symfony/event-subscriber
+        [
+            'main_type' => 'Symfony\Component\EventDispatcher\EventDispatcherInterface',
+            'collected_type' => 'Symfony\Component\EventDispatcher\EventSubscriberInterface',
+            'add_method' => 'addSubscriber',
+        ],
+    ];
+
+    public function __construct()
+    {
```

Comment:
Possibly use ctor config or detect Framework bundle to prevent duplicate command adding